jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -41,6 +41,15 @@ def status(self, repo_path):
 	def appraise_pull(self, git_root_dir):
 		self.run(git_root_dir, 'appraise', 'pull', self.remote)
 
+	def appraise_push(self, git_root_dir):
+		self.run(git_root_dir, 'appraise', 'push', self.remote)
+
+	def push_local_comments(self, git_root_dir):
+		self.run(git_root_dir, 'push', self.remote)
```

Comment:
I do not think we should be automatically pushing the user's local commits as part of this handler. Merely running `git appraise push` should be sufficient.